test/scripts/jenkins_xpack.sh

```diff
@@ -29,3 +29,9 @@ cd "$XPACK_DIR"
 node scripts/jest --ci --no-cache --verbose
 echo ""
 echo ""
+
+echo " -> Running jest integration tests"
+cd "$XPACK_DIR"
+node scripts/jest_integration --ci --no-cache --verbose
```

Comment:
After merging in master and resolving the 1 tiny conflict, I was able to run this in local and the tests passed! First time I've been able to see it do so :)